.github/workflows/main.yml

```diff
@@ -16,47 +16,6 @@ on:
   pull_request:
 
 jobs:
-  check:
```

Comment:
Should we change line 13 to:
name: PR Check CI (build)
and rename main.yml to build.yml?

Reply:
i want to make this sync with testing repo's main.yml for now.
maybe we can change it later.

Reply:
also, i'm not sure if build.yml is a better name. i guess we want to add a job to run eg. sim ostest sooner or later.

Reply:
> also, i'm not sure if build.yml is a better name. i guess we want to add a job to run eg. sim ostest sooner or later.

call it run.xml or test.xml?

Reply:
do you mean to have separate build.xml and test.xml?

Reply:
> do you mean to have separate build.xml and test.xml?

Maybe, if we can express the dependence across the yaml files, otherwise we need to stick into one yaml.

Reply:
see https://github.com/apache/incubator-nuttx-testing/pull/25
i will update this and apps once the testing one is merged.

Reply:
Should we name it build.xml? there isn't test at all.

Reply:
i wanted to save another rename. but i changed it to build.xml.